src/LoginRouter.js

```diff
@@ -198,6 +235,18 @@ function (BaseLoginRouter, PrimaryAuthController, VerifyDuoController, MfaVerify
       });
     },
 
+    enrollWindowsHello: function () {
+      this.render(EnrollWindowsHelloController, {
+        provider: 'FIDO',
+        factorType: 'webauthn',
+        Beacon: FactorBeacon
+      });
+    },
+
+    activateWindowsHello: function () {
+      this.controller.model.activate();
```

Comment:
as @rchild-okta commented before, this should render a controller. Take a look at his previous comment